### PR TITLE
docs: add an example notebook about line graphs

### PR DESCRIPTION
**Pull request by milkshakeiii**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb [hunk 1]
```diff
@@ -0,0 +1,602 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ffqBzbNztNvF"
+      },
+      "source": [
+        "### Dataset\n",
+        "\n",
+        "This notebook uses the [BigQuery COVID-19 Open Data](https://pantheon.corp.google.com/marketplace/product/bigquery-public-datasets/covid19-open-data)."
```
Comment (GarrettWu) [inline notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb:74]: Nit: suggest to talk a bit about the columns we select for the dataset.

File: notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+        "BigQuery DataFrames does not currently integrate with seaborn by default. So we will demonstrate how to downsample and download a DataFrame, and use seaborn on the downloaded data."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MmfgKMaEXNbL"
+      },
+      "source": [
+        "### Downsample and download"
```
Comment (GarrettWu) [inline notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb:433]: I thought we don't encourage our users to download the data?

File: notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "g4MeM8Oe9Q6X"
+      },
+      "outputs": [],
+      "source": [
+        "symptom_data = symptom_data[\n",
+        "    symptom_data[\"search_trends_cough\"].notna() &\n",
```
Comment (GarrettWu) [inline notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb:409]: this is hacky. A better approach would be df[[column0, column1]].dropna()

**Review by ashleyxuu — COMMENTED** [review]
@GarrettWu Since @milkshakeiii is on leave now, how about we check in the notebook, and edit it afterwards?

File: notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb (hunk 2 above)
Comment (milkshakeiii) [reply notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb:433]: Much of the time, yes, but downloading and downsampling for visualization was specifically requested for an example.

File: notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb (hunk 1 above)
Comment (milkshakeiii) [reply notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb:74]: Good idea, added, thanks.

File: notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb (hunk 3 above)
Comment (milkshakeiii) [reply notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb]: True, fixed, thanks.

**Comment by milkshakeiii** [comment]
> @GarrettWu Since @milkshakeiii is on leave now, how about we check in the notebook, and edit it afterwards?

It's okay, this is my last PR, I can respond to a few comments ! Thanks!

**Review by GarrettWu — APPROVED** [review]

**Review by ashleyxuu — APPROVED** [review]

**Pull request merged**